### ISSUE TITLE
Verify user email by admin

### ISSUE THREAD

**Issue by Mte90** [opensupports/opensupports#446]
I have an user with issues that cannot confirm his profile and as admin there is no button for it.
So what is the best way for it? I have to manually change the database?

**Comment by Mte90** [comment]
I see now that all the users are waiting for an approval, but I need that at registration is confirmed because I am using webhooks to register them.

**Comment by ivandiazwm** [comment]
Currently, they should have received an email with the confirmation link.
There's no way to do it from the admin panel. However, you can look them up in the database in the table `user` and change the value in the column `verification_token` to `NULL`. That would verify the user.

**Comment by Mte90** [comment]
Yes I did yesterday that but there is a way to avoid verification at registration?
I don't wan to do a cron that clean every `verification_token` every hour.

**Comment by Mte90** [comment]
I did a cron with https://gist.github.com/Mte90/0360d248b3e08a4e5c00633d1fcfdb2d

**Comment by nickbe** [comment]
I think someone (me? I'm not sure) already suggested that we should be able to verify a user from the user list. Especially as admin. I hope this is simple to implement and I would suggest to @ivandiazwm this little change.

**Comment by ivandiazwm** [comment]
Email verification is intended for ensuring the user has possession of that email so we can avoid spamming them and avoid identity theft.

If the verification email is not being received by the user, it means there's a problem with the SMTP configuration and probably all the other emails can neither be sent.

I think the best solution for this problem would be a "resend verification email" button both in the login and the admin view.

Also, keep in mind that if you remove all smtp configuration, no verification will be needed since emails won't be sent.

**Comment by nickbe** [comment]
I didn't mean that. The standard mail verification is perfect.
What I meant was that as administrator I should be able to add users directly mark them as verified.

**Comment by thiagojeffery** [comment]
Is there any way to resend the confirmation email?

**Comment by JCKodel** [comment]
+1
I use OpenSupports exclusively on API's for my users (all user interface is native inside a mobile app), only the staff uses the web UI, but I can't do that because there is no way to approve an user after signup (I already know his/her e-mail is valid because it came from Firebase (either Google, Facebook or validated e-mail/password, so, I don't need to bother my user again with it).

Besides, my OpenSupports is in maintenance mode (so users can't login through web UI), and the confirmation e-mail just shows the "Under maintenance" page =\ 

Solution: an extra argument in api/users/signup `"disableEmailValidation": 1`, so the user won't receive any e-mails.

**Comment by acostasf** [comment]
> +1
> I use OpenSupports exclusively on API's for my users (all user interface is native inside a mobile app), only the staff uses the web UI, but I can't do that because there is no way to approve an user after signup (I already know his/her e-mail is valid because it came from Firebase (either Google, Facebook or validated e-mail/password, so, I don't need to bother my user again with it).
> 
> Besides, my OpenSupports is in maintenance mode (so users can't login through web UI), and the confirmation e-mail just shows the "Under maintenance" page =\
> 
> Solution: an extra argument in api/users/signup `"disableEmailValidation": 1`, so the user won't receive any e-mails.

This solution will be nice for me
